src/Pages/ProductDetails/ProductDetails.tsx

```diff
@@ -1,14 +1,56 @@
+/* eslint-disable max-len */
 import { useEffect, useState } from 'react';
-import { useParams } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
 import { fetchProductById } from '../../api/products.api';
 import { Product } from '../../types/product';
+import classes from './productDetails.module.scss';
+import Arrow from '../../icons/Arrow';
+import Home from '../../icons/Home';
+import BackButton from '../../components/shared/buttons/BackButton';
 
 const ProductDetails: React.FC = () => {
   const { productId } = useParams();
 
   const [product, setProduct] = useState<Product | null>(null);
   const [error, setError] = useState<Error | null>(null);
 
+  const {
+    container,
+    grid,
+    icon,
+    title,
+    text,
+    link,
+    phone,
+    template,
+    info,
+    about,
+    button__back: buttonBack,
+    grid__desktop: gridDesktop,
+    grid__tablet: gridTablet,
+    grid__mobile: gridMobile,
+    grid__item: gridItem,
+    grid__item__mobile_1_2: gridMobileFullSize,
+    grid__item__tablet_1_6: gridTabletFullSize,
+    grid__item__desktop_1_8: gridDesktopFullSize,
+    grid__item__desktop_1_4: gridDesktopPhotoBlock,
+    grid__item__desktop_5_8: gridDesktopInfoBlock,
+    grid__item__tablet_1_4: gridTabletPhotoBlock,
+    grid__item__tablet_5_8: gridTabletInfoBlock,
+    grid__item__desktop_1_4: gridDesktopAboutBlock,
+    grid__item__desktop_5_8: gridDesktopTechSpecsBlock,
```

Comment:
in our website we have 24 grid column, so: 
```suggestion
    grid__item__mobile_1_4: gridMobileFullSize,
    grid__item__tablet_1_12: gridTabletFullSize,
    grid__item__desktop_1_24: gridDesktopFullSize,
    grid__item__desktop_1_12: gridDesktopPhotoBlock,
    grid__item__desktop_13_24: gridDesktopInfoBlock,
    grid__item__tablet_1_7: gridTabletPhotoBlock,
    grid__item__tablet_8_12: gridTabletInfoBlock,
    grid__item__desktop_1_12: gridDesktopAboutBlock,
    grid__item__desktop_13_24: gridDesktopTechSpecsBlock,
```